packages/UsageAnalysis/files/Test Track/Queries/Advanced/Visual Query.md

```diff
@@ -1,16 +0,0 @@
-1. Go to **Browse** > **Databases**.
-2. Expand **PostgresDart** > **northwind** > **Schemas** > **public**. A list of tables opens.
-3. Right-click the **products** DB table and select **Visual Q**uery from the context menu.
```

Comment:
Do we have autotests for visual query? it sounds like UI thing which should be tested manually, but i can be wrong

Reply:
Pavlo told me to add "New Agreg. query" test scenario instead of VisualQuery